Dockerfile.jupyter

```diff
@@ -7,7 +7,7 @@ FROM openjdk:17-slim
 ##
 
 ## `wget` is used to retrieve Conda and SysML Release. Inkscape and
-## LaTeX is required for rendering notebooks as PDFs.
+## LaTeX are required for rendering notebooks as PDFs.
```

Comment:
LaTeX is plural?

Reply:
LaTeX is not plural. "Inkscape and LaTeX" are, though.